src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentCost = 40;
+  const threeDaysDiscount = 20;
+  const sevenDayDiscount = 50;
+  const fullRentCost = dailyRentCost * days;
+
+  if (days < 3) {
+    return fullRentCost;
+  } else if (days >= 3 && days < 7) {
+    return fullRentCost - threeDaysDiscount;
+  } else {
+    return fullRentCost - sevenDayDiscount;
+  }
```

Comment:
If you add retunr inside if block else redundant
```suggestion
  if (days < 3) {
    return fullRentCost;
  }
  
  if (days >= 3 && days < 7) {
    return fullRentCost - threeDaysDiscount;
  }
 
  return fullRentCost - sevenDayDiscount;
 
```